### DISCUSSION TITLE
Unable to compile file, unable to start project

### DISCUSSION THREAD

**Discussion by haohao-Ct** [Sixze/ALS-Refactored#549] [Q&A]
I'm trying to open the project file in ue5.4cpp and it shows that the following modules are missing or built with a different engine version:ALSALSCamera ALSExtrasALSEditorWould you like to rebuild them now I've tried a lot of ways, VS compiler 2022 is also uninstalled,In reinstallation,Or the compilation fails to start the project,Need to rebuild,I still hope that the big guy can come out with a tutorial on how to compile,Of course, I hope you can help me solve this problem,Thank you very much!

**Comment by GStreetGames** [comment]
Simply follow the instructions in the quick start section.
https://github.com/Sixze/ALS-Refactored?tab=readme-ov-file#quick-start

**Reply by lyc233nb** [reply]
did exactly the same as quick start. It keeps saying try comiple mannually. Tries tons of times genrating visual studio files. Put it in the Plugins folder.  I also tried tons of times building from visual studio. why it is so difficult to use?
